documentation/deployments.md

```diff
@@ -49,6 +49,13 @@ When a PR is approved and merged into `master` branch, the GitHub actions workfl
 - Calls the [deploy_app.yml](/.github/workflows/deploy_app.yml) workflow to use Terraform to update the `web` and `worker` apps to use the new Docker image, and apply any changes to the `production` environment
 - Sends a Slack notification to the `#twd_tv_dev` channel
 
+### Merge and concurrency deployment management
+When there are multiple merges, this could lead to raise conditions. To mitigate this, the `turnstyle` action prevents two or more instances of the same workflow from running concurrently.
+
+Furthermore, to help with code reuse we trigger a separate deployment workflow via the workflow dispatch action. We must pass the `ref` (branch) or pull request `ref` to locate the workflow.
```

Comment:
What does the workflow dispatch action do? I'm not sure I understand how it helps with code reuse.

To be consistent, let's say `workflow_dispatch` (with backticks `) rather than 'workflow dispatch'.

Reply:
workflow_dispatch: allows manual trigger of workflows via the UI and programtically.

The deployment workflows all share\reuse `deploy_app` workflow, which contains the terraform code logic for deploying `sha` commits to the environment(s).

Reply:
> The deployment workflows all share\reuse deploy_app workflow, which contains the terraform code logic for deploying sha commits to the environment(s).

This is helpful context - may be worth adding in brackets?

Reply:
Unless you think this is likely to change, which would make the documentation out of date.

Reply:
@david-mears-dfe we made it more generic with `workflow code reuse` as it applies to terraform, but also smoke tests or any reusable workflow